RallyHereDebugTool/Source/Public/RallyHereDebugTool.h

```diff
@@ -115,6 +115,8 @@ class RALLYHEREDEBUGTOOL_API URallyHereDebugTool : public UGameInstanceSubsystem
 	TSharedPtr<struct FRHDTW_Analytics> AnalyticsWindow;
 	TSharedPtr<struct FRHDTW_About> AboutWindow;
 
+	TMap<FString, bool> SavedWindowVisibilities;
```

Comment:
We put the flags for this into the window base class before - that way it could be passed to the window function, which is what generates the `X` close button on each window.  That is probably worth putting back for consistency

Reply:
This array was present previously (it was how we saved off visibility between frames).  The flags on the windows were just synced to it every tick.

Reply:
Specifically, this was previously a UPROPERTY(config) to persist across runs.  However, we have moved where we saved config for imgui since, so I did not want to restore that part of the feature.